src/txn/txn.c

```diff
@@ -1664,8 +1664,15 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
                 if (!F_ISSET(op, WT_TXN_OP_KEY_REPEATED))
                     WT_ERR(__txn_resolve_prepared_op(session, op, true, &cursor));
 
-                /* Sleep between resolving prepared operations when configured. */
-                __wt_timing_stress(session, WT_TIMING_STRESS_PREPARE_RESOLUTION, NULL);
+                /*
+                 * Sleep for some number of updates between resolving prepared operations when
+                 * configured, however, avoid causing too much stress when there are a large number
+                 * of updates. Multiplying by 36 provides a reasonable chance of calling the stress
+                 * without exceeding a total of 36 calls over the total mod_count.
+                 */
+                if ((i * 36) % txn->mod_count == 0) {
```

Comment:
There is no need for `{}` as it is a single statement.

Reply:
Fixed.